docs/reference_guides/core/costing/costing_framework.rst

```diff
@@ -0,0 +1,70 @@
+IDAES Process Costing Framework
+===============================
+
+.. contents:: Contents 
+    :depth: 4
+
+Introduction
+------------
+
+.. rubric:: Available Costing Libraries
+
+This document outlines IDAES support for general process costing and integrating custom costing packages into IDAES models. The base costing framework supports attaching externally-defined blocks and methods for flowsheet and unit model cost calculations. Users must provide a costing package to use for cost calculations; IDAES provides a set of preexisting :ref:`Available Costing Libraries<reference_guides/core/costing/costing_libraries:Available Costing Libraries>`.
```

Comment:
Add " for use as examples and guides users in developing their own costing packages." to the end of this to help make it clear that users can add their own.